monkey/monkey_island/cc/environment/test_environment_config.py

```diff
@@ -22,6 +22,7 @@
 STANDARD_WITH_CREDENTIALS = os.path.join(
     TEST_RESOURCES_DIR, "server_config_standard_with_credentials.json"
 )
+DATA_DIR = os.path.join(TEST_RESOURCES_DIR, "server_config_data_dir.json")
```

Comment:
This is a path to a file `server_config_data_dir.json`. Name `DATA_DIR` implies that it's a directory full of some kind of data. Find a more specific name for this variable, that would convey what `server_config_data_dir.json` is, like `PATH_TO_SOMETHING`